pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -122,9 +125,25 @@ func Platform() (*baremetal.Platform, error) {
 		ProvisioningNetworkCIDR:      parsedCIDR,
 		ProvisioningNetworkInterface: provisioningNetworkInterface,
 		Hosts:                        hosts,
+		APIVIPs:                      []string{apiVIP},
+		IngressVIPs:                  []string{ingressVIP},
```

Comment:
These appear to be set correctly provided that the DNS records are set up already (which IIRC is required before you actually create the cluster):
https://github.com/openshift/installer/blob/master/pkg/types/baremetal/defaults/platform.go#L126-L138

Setting them here would override that behaviour, which we don't want.